src/vs/workbench/contrib/issue/electron-sandbox/issue.contribution.ts

```diff
@@ -19,6 +19,9 @@ import { IDialogService } from 'vs/platform/dialogs/common/dialogs';
 import { INativeHostService } from 'vs/platform/native/common/native';
 import { IProgressService, ProgressLocation } from 'vs/platform/progress/common/progress';
 import { IIssueMainService, IssueType } from 'vs/platform/issue/common/issue';
+// eslint-disable-next-line local/code-import-patterns
```

Comment:
wondering if location here is incorrect. getting import violation errors

Reply:
You need to put it in a common platform location